catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/taxonomy/MediaAttributes.java

```diff
@@ -0,0 +1,127 @@
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.data.impl.taxonomy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import ddf.catalog.data.AttributeDescriptor;
+import ddf.catalog.data.impl.AttributeDescriptorImpl;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.taxonomy.Media;
+import ddf.catalog.data.taxonomy.MetacardComponent;
+
+/**
+ * This class provides attributes that reflect metadata about media resources associated with a
+ * metacard.  This includes bit rates, encoding and compression.
+ * <p>
+ * https://codice.atlassian.net/wiki/display/DDF/Attributes
+ */
+public class MediaAttributes implements Media, MetacardComponent {
+
+    private static final Set<AttributeDescriptor> DESCRIPTORS = new HashSet<>();
+
+    private static final String NAME = "media";
+
+    static {
+        DESCRIPTORS.add(new AttributeDescriptorImpl(FORMAT,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(FORMAT_VERSION,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(BIT_RATE,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(BITS_PER_SAMPLE,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(COMPRESSION,
+                true /* indexed */,
+                true /* stored */,
+                false /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.INTEGER_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(ENCODING,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(FRAME_CENTER,
+                true /* indexed */,
+                true /* stored */,
+                true /* tokenized */,
+                true /* multivalued */,
+                BasicTypes.STRING_TYPE));
+        DESCRIPTORS.add(new AttributeDescriptorImpl(FRAME_RATE,
```

Comment:
This will always be fps, so it would make sense to change this to `FRAMES_PER_SECOND`